src/Article.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const Article = ({ title, text, date }) => (
+  <div className="wrapper">
+    <h1>
+      {title}
+    </h1>
```

Comment:
Don't need to add extra lines, code is short enough.
```suggestion
    <h1>{title}</h1>
```